src/scribe_data/cli/main.py

```diff
@@ -127,7 +128,11 @@ def main() -> None:
     )
     total_parser._actions[0].help = "Show this help message and exit."
     total_parser.add_argument(
-        "-lang", "--language", type=str, help="The language(s) to check totals for."
+        "-l",
```

Comment:
I'm going to remove `-l` as an option, @mhmohona, as I think that making sure that we don't have commands and arguments that are the same is important :)